src/test/sql/sql_pipeline_statement_test.cpp

```diff
@@ -57,6 +57,7 @@ class SQLPipelineStatementTest : public BaseTest {
     _multi_statement_parse_result = std::make_shared<hsql::SQLParserResult>();
```

Comment:
I can’t add this to firstNames/lastNames.txt, because I am offline and Github didn’t load them:
As said above, I do not think that we should bring in the playground (which is the only part using these files). Even if we did, we would have to resort to generating some other data. A 26k line dummy file has no place in a code repository.